src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/ImplementIDisposableCorrectly.cs

```diff
@@ -543,75 +585,180 @@ private bool ValidateExpression(IExpressionStatementOperation expressionStatemen
                 return false;
             }
 
-            private bool IsDisposeBoolCall(IInvocationOperation invocationExpression)
+            private bool IsSuppressFinalizeCall(IInvocationOperation invocationExpression)
```

Comment:
Is this method similar to the previous implementation or are there any changes?

Reply:
The diff here isn't particularly enlightening. IsDisposeBoolCall was actually moved up above, as both VerifyFinalizer and VerifyDispose needed it. I didn't make any changes to the original IsSuppressFinalize call. The only change to IsIsDisposeBool was making the class variable access be a parameter.

---
In reply to: [151559008](https://github.com/dotnet/roslyn-analyzers/pull/1371#discussion_r151559008) [](ancestors = 151559008)